provider/internal/device/ios/resource.go

```diff
@@ -108,6 +110,14 @@ func (r *IosDeviceResource) Schema(ctx context.Context, req resource.SchemaReque
 				MarkdownDescription: "Set this attribute to true if you do not want CDO to validate the certificate of this device before onboarding.",
 				Required:            true,
 			},
+			"tags": schema.ListAttribute{
+				MarkdownDescription: "The tag associated with the device.",
```

Comment:
I was just reading the CDO documentation: we call tags `labels` in the documentation. So let's call them `labels` here in the provider (let's continue to call them tags in the client, because that's what they are internally). 

Could you also please change the markdown description to: "Set a list of labels to identify the device as part of a group. Refer to the [CDO documentation](https://docs.defenseorchestrator.com/t-applying-labels-to-devices-and-objects.html#!c-labels-and-filtering.html) for details on how labels are used in CDO."

Reply:
Sure, done